src/components/molecules/DateRangePicker/DateRangePicker.tsx

```diff
@@ -0,0 +1,103 @@
+import DatePickerInput from 'components/molecules/DatePickerInput/DatePickerInput'
+import { Ref, forwardRef, useCallback } from 'react'
+import { FieldError } from 'react-hook-form'
+import classNames from 'classnames'
+import { ReactComponent as Delete } from 'assets/icons/delete.svg'
+import classes from './classes.module.scss'
+import { useTranslation } from 'react-i18next'
+import Button, { AppearanceTypes, IconPositioningTypes } from '../Button/Button'
```

Comment:
**[non-blocker]** Not too important but we should use absolute imports everywhere